hibike/hibikeDevices.json

```diff
@@ -67,6 +61,26 @@
                                                      ]
     },
 
+
+    {"id": 10,     "name": "YogiBear",      "params": [   
+                                                          {"number": 0  , "name": "enable"              , "type": "bool"     , "read": true , "write": true  }, 
+                                                          {"number": 1  , "name": "command_state"       , "type": "uint8_t"  , "read": true , "write": true  },
+                                                          {"number": 2  , "name": "duty_cycle"          , "type": "float"    , "read": true , "write": true  },
+                                                          {"number": 3  , "name": "pid_pos_setpoint"    , "type": "float"    , "read": true , "write": true  },
+                                                          {"number": 4  , "name": "pid_pos_kp"          , "type": "float"    , "read": true , "write": true  },
+                                                          {"number": 5  , "name": "pid_pos_ki"          , "type": "float"    , "read": true , "write": true  },
```

Comment:
PID constants are listed as readable but the c++ code to read them returns 0. One of these should be changed.